public/gameStates/pixelArtRoom.js

```diff
@@ -40,22 +40,27 @@ class PixelArtRoom {
 
     initListeners() {
         new KeyListener(
-            'ArrowUp',
+            [ 'ArrowDown', 'KeyJ' ],
+            () => (this.localPlayer.velY = 0),
```

Comment:
What should this do? Because at the moment it only partly stops the player mid-air, when key is pressed long enough